src/main/mainWindow.js

```diff
@@ -67,6 +67,7 @@ function createMainWindow(config, options) {
 
   mainWindow.webContents.on('will-attach-webview', (event, webPreferences) => {
     webPreferences.nodeIntegration = false;
```

Comment:
Now it's configured in `app.on('web-contents-created')` event. So I think this part is no longer needed.

Reply:
Maybe not, but I want to keep it just to make sure.